content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,89 @@
+=============
+
+Putaway is the process of receiving shipments and routing the products in them to the appropriate
+storage locations. This process can be accomplished seamlessly in Odoo using *putaway rules*.
+
+The rules detail how products move into and throughout specified locations in the warehouse.
+They are triggered by the arrival of shipments in a certain location. The automatically generated
+rules coordinate efficient inter-warehouse transfers.
 
-A good warehouse implementation takes care that products automatically move to their appropriate
-destination location. To make that process easier, Odoo uses *Putaway rules*. But what is a putaway
-rule? Putaway is the process of taking products off the receiving shipments and directly putting
-them into the most appropriate location.
+If, for example, a warehouse contains multiple volatile substances that should not come into
+contact, it is crucial that these substances not be stored in close proximity to one another. Using
+putaway rules, it is possible to ensure that such products are stored in different locations.
 
-If, for example, a warehouse contains volatile substances, it is important to make sure that certain
-products are not stored close to each other because of a potential chemical reaction. That’s where
-putaway rules intervene, to avoid storing products wrongly.
+.. seealso::
+   - :ref:`Routes-- how products move<inventory/routes/use_routes/push>`
+   - :ref:`Warehouse locations<inventory/warehouse_location/definition>`
 
 Configuration
 =============
 
-In the *Inventory* app, go to :menuselection:`Configuration --> Settings` and activate the
-*Multi-Step Routes*. By doing so, the *Storage Locations* will be automatically activated.
+Begin by navigating to :menuselection:`Inventory --> Configuration --> Settings`, then activate the
+:guilabel:`Multi-Step Routes` checkbox under the :guilabel:`Warehouse` heading. By doing so, the
+:guilabel:`Storage Locations` setting is automatically enabled as well. Finally, click
+:guilabel:`Save`.
 
-.. image:: putaway/putaw1.png
+.. image:: putaway/multi-step-routes.png
    :align: center
+   :alt: Check multi-step routes box
 
-Setting up a Putaway Rule
-=========================
+Create a putaway rule
+=====================
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
-different locations to maintain product quality.
+To manage where specific products are routed for storage, navigate to :menuselection:`Inventory -->
+Configuration --> Putaway Rules`. Then, click on :guilabel:`Create` to configure a new putaway rule.
+Choose the :guilabel:`Product` and/or :guilabel:`Product Category` that the rule will affect. Set
+:guilabel:`When product arrives in` as the :ref:`location<inventory/warehouse_location/definition>`
+where the rule will be triggered and :guilabel:`Store to` as the location where products affected by
+the rule will be stored. Finally, click :guilabel:`Save`.
```

Comment:
```suggestion
the rule will be stored. Finally, click :guilabel:`Save`.
```

For these choices here, it could help the reader to know why in this instance we want to configure it the way you're telling them to. For example, why are we using specific product categories or locations in the WH at all?